app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -19,16 +19,25 @@ def self.raw_create_volume(ext_management_system, options = {})
 
     if options['mode'] == 'Basic'
       creation_hash[:service] = ext_management_system.storage_services.find(options["storage_service_id"]).ems_ref
+      if options["count"] == 0
+        target_class = :cloud_volumes
+        target_option = "new"
+      else
+        target_class = nil
+        target_option = "ems"
```

Comment:
Follow-up, but we can do targeted refresh of multiple targets which will still be way more efficient than a full ems refresh.  Can you get the list of ems_refs of the new volumes?

Reply:
the advanced volume create leads to the backend first creating a new service, attaching it to the selected resources and creating the volume(s) on one of the resources. so we get the `ems_ref`s of the new volume(s), but not that of the new service and `storage_service_resource_attachments`.

if triggering a full refresh is too much of an overkill, we'll leave it as targeted for what we can get, and we'll just have to manually perform a full refresh until we manage to get `ems_ref`s for various types of objects in one job.